src/domain/event/utils.ts

```diff
@@ -13,3 +18,70 @@ export const getEventPlaceholderImage = (id: string): string => {
 
   return EVENT_PLACEHOLDER_IMAGES[index];
 };
+
+/**
+ * Get first availanle language of an event
```

Comment:
btw why do we need these docs?

Reply:
Maybe we don't. I'm just used to write them